lldb/source/Plugins/SymbolFile/DWARF/SymbolFileDWARF.cpp

```diff
@@ -4349,6 +4349,13 @@ LanguageType SymbolFileDWARF::GetLanguageFamily(DWARFUnit &unit) {
   return LanguageTypeFromDWARF(lang);
 }
 
+double SymbolFileDWARF::GetSymbolDownloadTime() {
+  double time = GetObjectFile()->GetFileSpec().GetDownloadTime();
+  if (m_dwp_symfile)
+    time += m_dwp_symfile->GetObjectFile()->GetFileSpec().GetDownloadTime();
```

Comment:
We should add a `ElapsedTime` member variable to ObjectFile.h instead of adding to FileSpec.h. `ObjectFile` objects already have many instance variables and it is ok to add the size to that class, but not to `FileSpec`.